frontend/src/scenes/scenes.ts

```diff
@@ -130,7 +130,7 @@ export const sceneConfigurations: Partial<Record<Scene, SceneConfig>> = {
 }
 
 export const redirects: Record<string, string | ((params: Params) => string)> = {
-    '/': urls.insights(),
+    '/': urls.savedInsights(),
```

Comment:
This change is interesting. I'm a bit sad that exploration is now less of a default, though this is probably better for the collaboration angle.

Reply:
We will probably want to test this a bit more ongoingly, I'm particularly worried it will negatively impact new users who just signed up to an account. Our empty state here needs to be killer. Thoughts @clarkus ?